### ISSUE TITLE
Balance optimization 

### ISSUE THREAD

**Issue by gcperk** [bcgov/PEMmodelr#4]
Testing notes for @kdaust 
- missing function report_model_accuracy(). Is this the old acc_metrix function? **FOUND and added to modelr**
- 
-

**Comment by kdaust** [comment]
Yes it is - which probably needs updating but was working when I tested last
